postalicious/public/main.js

```diff
@@ -0,0 +1,244 @@
+
+      let currentTimePL = Date.now()      
+      let minutesRender = Math.floor(Math.abs(Math.floor((timeBefore-currentTimePL) / 1000) / 60))
+      let secondsRender = Math.floor(Math.abs(Math.floor((timeBefore-currentTimePL) / 1000) % 60))
+      timeDifference[1] = 'Minutes: ' + minutesRender + ', Seconds: ' + secondsRender
+    }
+    let bodyContainer = generateParent(responseWindow)
+    bodyContainer.style.minHeight = '350px'
+    generateElement('bold', 'body:', bodyContainer, 1)
+    generateElement('standard', domValues.body, bodyContainer, 5, true)
+
+    let timeToRespondAJAX = generateParent(responseWindow)
+    generateElement('bold', 'time taken for response:', timeToRespondAJAX, 1)
+    generateElement('standard', timeDifference[0], timeToRespondAJAX, 5, true) 
+
+    let timeToRespondPL = generateParent(responseWindow)
+    generateElement('bold', 'time taken for render:', timeToRespondPL, 1)
+    generateElement('standard', timeDifference[1], timeToRespondPL, 5, true)
+
+    let httpVersionContainer = generateParent(responseWindow)
+    generateElement('bold', 'http version:', httpVersionContainer, 1)
+    generateElement('standard', domValues.httpVersion, httpVersionContainer, 5)
+
+    let statusCodeContainer = generateParent(responseWindow)
+    generateElement('bold', 'status code:', statusCodeContainer, 1)
+    generateElement('standard', domValues.statusCode, statusCodeContainer, 5)
+
+    generateElement('bold', 'response Headers:', responseWindow, 0)
+    let headerParentContainer = generateParent(responseWindow)
+    headerParentContainer.style.flexDirection = 'column'
+    headerParentContainer.style.height = '400px'
+    if('headers' in domValues) {
+      for( let headerKey in domValues.headers) {
+        let headerParent = generateParent(headerParentContainer)
+        generateElement('bold', headerKey+':', headerParent, 1)
+        headerParent.lastChild.style.left = '10px'
+        generateElement('standard', domValues.headers[headerKey], headerParent, 5)
+      }
+    }
+  }
+}
+
+function generateElement(type, content, parent, grow, overflow) {
+  let element = document.createElement('p')
+  if(type === 'bold') {
+    element.style.fontWeight = '700'
+  }
+  // TODO Fix styling issue when too much text is displayed in headers and body - overlap occurs
+  element.style.flexGrow = grow !== undefined ? grow : 1
+  element.className = 'requestResponseNode'
+  if(overflow) {
+    element.className = element.className + ' overflow'
+  }
+  element.textContent = content
+  parent.appendChild(element)
+}
+
+function generateParent(mainWindow) {
+  let parent = document.createElement('div')
+  parent.className = 'requestResponseParent'
+  mainWindow.appendChild(parent)
+  return parent
+}
+
+function cleanDom(windowToClean) {
+  while (windowToClean.hasChildNodes()) {
+    windowToClean.removeChild(windowToClean.lastChild)
+  }
+}
+
+window.addEventListener('load', () => { 
```

Comment:
__IMPORTANT__: You should be using the DOMReady event NOT the load event. The load event takes forever when you have other assets like images and videos.